### DISCUSSION TITLE
How to fix the cross in nuxt.js with Laravel?

### DISCUSSION THREAD

**Discussion by michaelnabil230** [laravel/framework#42020] [Q&A]
Hello, everybody, I have one problem. I need help
When calling API with Axios and adding in headers `Authorization`  or ex `X-Tanent` make two requests one with request code 204 and the method `OPTIONS` and another request code 200 and the method `GET`

<img width="1723" alt="Screen Shot 2022-04-17 at 4 13 27 PM" src="https://user-images.githubusercontent.com/46572405/163718287-f62dcaf4-b06c-47b6-be9d-8e212ea29b0c.png">
<img width="564" alt="Screen Shot 2022-04-17 at 4 15 10 PM" src="https://user-images.githubusercontent.com/46572405/163719658-1f078d6b-4019-45b2-836d-dad6d228652b.png">
<img width="564" alt="Screen Shot 2022-04-17 at 4 15 34 PM" src="https://user-images.githubusercontent.com/46572405/163719661-68e1ccc9-0c84-4de0-9619-9862ed447f9b.png">

```vue

<script>
export default {
  data() {
    return {
      data: [],
    };
  },
  mounted: function () {
    console.log("mounted");
    this.$axios
      .get(url + "/V1/brands", {
        headers: {
          Authorization: "Bearer token",
          "X-Tanent": "tanent",
        },
      })
      .then((res) => {
        this.data = res.data.brands.data;
      });
  },
};
</script>
```

**Comment by P-u-r-i-a** [comment]
Please have a look at the following link (Mozilla docs):

https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request

Regards,
Puria
